ext/crypto/lib.rs

```diff
@@ -272,12 +273,21 @@ pub enum KeyFormat {
   Pkcs8,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum KeyType {
+  Private,
+  Public,
+  Secret,
+}
+
 #[derive(Deserialize)]
 #[serde(rename_all = "lowercase")]
```

Comment:
```suggestion
#[serde(rename_all = "camelCase")]
```

Minor nit. All `key_type` in JS side can now be renamed to `keyType`.